### PR TITLE
Lesson 5

### PR DESCRIPTION
**Pull request by olennikovandrey**

### CODE REVIEW

File: js/lesson-5.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+console.log("lesson_5_homework");
+//point 5 of homework
+let date = "2020-11-26"; //a string variable is set
+function formattedDate() {
+    let dateSplit = date.split("-"); //split a string variable into an array by "-" using the "split" method
+    let newDateView = (dateSplit[2] + "." + dateSplit[1] + "." + dateSplit[0]); //set a variable to combine array elements in a given sequence
+    return newDateView
```
Comment (Dronom) [inline js/lesson-5.js:7]: where is`;`?

File: js/lesson-5.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    },
+    {
+    country: "Germany",
+    city: "Berlin",
+    hotel: "Hotel Rehberge Berlin Mitte",
+    },
+];
+
+const searchCountryInfo = str => {
+  let result = [];
```
Comment (Dronom) [inline js/lesson-5.js:57]: const!!

Comment (Dronom) [reply js/lesson-5.js:57]: 😠

Comment (Dronom) [reply js/lesson-5.js:57]: when you initialize an array or object, always use const

File: js/lesson-5.js (hunk 1 above)
Comment (olennikovandrey) [reply js/lesson-5.js:7]: fixed

File: js/lesson-5.js (hunk 2 above)
Comment (olennikovandrey) [reply js/lesson-5.js:57]: fixed, last time, I promise) arr = const, ok)

File: js/lesson-5.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+console.log("lesson_5_homework");
+//point 5 of homework
+let date = "2020-11-26"; //a string variable is set
```
Comment (Dronom) [inline js/lesson-5.js:3]: what about it? do you change it?

File: js/lesson-5.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+function formattedDate() {
+    let dateSplit = date.split("-"); //split a string variable into an array by "-" using the "split" method
+    let newDateView = (dateSplit[2] + "." + dateSplit[1] + "." + dateSplit[0]); //set a variable to combine array elements in a given sequence
```
Comment (Dronom) [inline js/lesson-5.js:6]: and it

File: js/lesson-5.js (hunk 3 above)
Comment (olennikovandrey) [reply js/lesson-5.js:3]: fixed to const

File: js/lesson-5.js (hunk 4 above)
Comment (olennikovandrey) [reply js/lesson-5.js:6]: and it)

**Pull request closed without merging**